rom/laghos_rom.hpp

```diff
@@ -683,8 +686,9 @@ class ROM_Operator : public TimeDependentOperator
 
     mutable double dt_est_SP = 0.0;
 
-    bool useReducedMv = false;  // TODO: remove this?
-    bool useReducedMe = false;  // TODO: remove this?
+    bool sns1 = false;
```

Comment:
The meaning of SNS Type 1 should be documented somewhere.

Reply:
@dylan-copeland A comment is added and the corresponding equation in the SNS paper is mentioned.